### PR TITLE
docs: update README.md

### PR DESCRIPTION
**Pull request by SvetoslavTsenov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,21 +2,53 @@
 
 A package to help with writing and executing e2e [Appium](http://appium.io) tests.
 
-## <a name='usage'></a>Usage
+## <a name='requirements'></a>Requirments
 
-Install it with:
+The `nativescript-dev-appium` plugin requires:
+* latest version of [Appium](https://github.com/appium/appium/releases)
```
Comment (zbranzov) [inline README.md]: Which is the latest version? I mean now is 'x' but tomorrow will the new one be compatible with the plugin. Maybe we should commit to current one.

File: README.md [hunk 2]
```diff
@@ -2,21 +2,53 @@
...
+                ├── appium.capabilities.json
+                ├── mocha.opts
+            ├── sample.e2e-test.ts
+            ├── setup.ts
+            ├── tsconfig.json
+        ├── ...
+        ├── package.json
+        ├── tsconfig.json
+
+> Note: To avaid any incompatibilities between the source of `e2e` tests and the source of the application, we recommend to exclude the `e2e` folder from the application's `tsconfig.json` file: `exclude": [ "e2e" ]`
```
Comment (zbranzov) [inline README.md]: Typo 'avoid'. Not sure why I have to exclude the e2e folder.

File: README.md [hunk 3]
```diff
@@ -126,17 +161,30 @@ $ npm run e2e --runType android25 --sauceLab --appLocation demo.apk --capsLocati
 
 ```
 
+## Features
+
+1. Compare images. Block out areas to ignore in comparison.
+2. Find elements findElementByText, findElementsByXPath, findElementByAccessibilityId etc...
+3. Gesture support: swipe, scroll, drag, scrollTo
+4. Action support: tap, click, doubleTap, hold, 
+5. Element characteristics: lcation, exists, size, isDisplayed
```
Comment (zbranzov) [inline README.md]: typo 'location'

File: README.md [hunk 4]
```diff
@@ -126,17 +161,30 @@ $ npm run e2e --runType android25 --sauceLab --appLocation demo.apk --capsLocati
...
+6. Find strategies: waitForExist, waitForExistNot
+7. Direct access to webdriver and webdriverio
+
 ## Troubleshooting
 
 Use the `--verbose` option to get error details:
 
 ```
-$ npm run appium --runType=android --verbose
+$ npm run appium --runType android25 --verbose
```
Comment (zbranzov) [inline README.md]: npm run e2e -- --runType android25 --verbose

File: README.md [hunk 5]
```diff
@@ -126,17 +161,30 @@ $ npm run e2e --runType android25 --sauceLab --appLocation demo.apk --capsLocati
 
 ```
```
Comment (zbranzov) [inline README.md]: npm run e2e -- --runType android25 --sauceLab --appPath demo.apk --capsLocation "/e2e-tests/config"

Comment (vchimev) [reply README.md]: Done.

File: README.md (hunk 1 above)
Comment (vchimev) [reply README.md]: We commit that *the new one* will be compatible with the plugin and avoid a specific version on purpose.

**Pull request merged**